mmv1/products/blockchainnodeengine/BlockchainNodes.yaml

```diff
@@ -0,0 +1,210 @@
+        properties:
+          - !ruby/object:Api::Type::String
+            name: jsonRpcApiEndpoint
+            output: true
+            description: |
+              The assigned URL for the node JSON-RPC API endpoint.
+          - !ruby/object:Api::Type::String
+            name: websocketsApiEndpoint
+            output: true
+            description: |
+              The assigned URL for the node WebSockets API endpoint.
+  - !ruby/object:Api::Type::NestedObject
+    name: ethereumDetails
+    description: |
+      User-provided key-value pairs
+    properties:
+      - !ruby/object:Api::Type::NestedObject
+        name: gethDetails
+        description: |
+          User-provided key-value pairs
+        properties:
+          - !ruby/object:Api::Type::Enum
+            name: garbageCollectionMode
+            immutable: true
+            values:
+              - :GARBAGE_COLLECTION_MODE_UNSPECIFIED
+              - :FULL
+              - :ARCHIVE
+            description: |
+              Blockchain garbage collection mode.
+      - !ruby/object:Api::Type::NestedObject
+        name: additionalEndpoints
+        output: true
+        description: |
+          User-provided key-value pairs
+        properties:
+          - !ruby/object:Api::Type::String
+            name: beaconApiEndpoint
+            output: true
+            description: |
+              The assigned URL for the node's Beacon API endpoint.
+          - !ruby/object:Api::Type::String
+            name: beaconPrometheusMetricsApiEndpoint
+            output: true
+            description: |
+              The assigned URL for the node's Beacon Prometheus metrics endpoint.
+          - !ruby/object:Api::Type::String
+            name: executionClientPrometheusMetricsApiEndpoint
+            output: true
+            description: |
+              The assigned URL for the node's execution client's Prometheus metrics endpoint.
+      - !ruby/object:Api::Type::Enum
+        name: network
+        immutable: true
+        values:
+          - :NETWORK_UNSPECIFIED
+          - :MAINNET
+          - :TESTNET_GOERLI_PRATER
+          - :TESTNET_SEPOLIA
+        description: |
+          The Ethereum environment being accessed.
+      - !ruby/object:Api::Type::Enum
+        name: nodeType
+        immutable: true
+        values:
+          - :LIGHT
+          - :FULL
+          - :ARCHIVE
+        description: |
+          The type of Ethereum node.
+      - !ruby/object:Api::Type::Enum
+        name: executionClient
+        immutable: true
+        values:
+          - :EXECUTION_CLIENT_UNSPECIFIED
+          - :GETH
+          - :ERIGON
+        description: |
+          The execution client
+      - !ruby/object:Api::Type::Enum
+        name: consensusClient
+        immutable: true
+        values:
+          - :CONSENSUS_CLIENT_UNSPECIFIED
+          - :LIGHTHOUSE
+          - :ERIGON_EMBEDDED_CONSENSUS_LAYER
```

Comment:
This option is [described as deprecated in the documentation](https://cloud.google.com/blockchain-node-engine/docs/reference/rest/v1/projects.locations.blockchainNodes#ConsensusClient:~:text=ERIGON_EMBEDDED,GitHub%20for%20details.) and "this option is not currently available when creating new blockchain nodes". Perhaps we should exclude it from the resource, to avoid user confusion when trying to make new nodes?